### DISCUSSION TITLE
CRUD Table Editing - Making edit window big as the whole cell

### DISCUSSION THREAD

**Discussion by tarasowski** [KevinVandy/mantine-react-table#97] [Q&A] [answered]
Hey everyone,
is there a way to make the edit window bigger so it fits the whole cell?
Right now it is very small, it would be great to have it bigger.

![Screenshot 2023-07-21 at 12 38 05](https://github.com/KevinVandy/mantine-react-table/assets/33748291/128b8297-e98c-43d1-9bb4-4a56ddc3c8cd)

Thanks,
Dimi

**Comment by KevinVandy** [comment, marked as answer]
you can customize the text box with `mantineEditTextInputProps` per column. Add some CSS. However, you might want to use a custom `Edit` component using the Mantine [`Textarea`](https://mantine.dev/core/textarea/) component instead. Maybe this will be a built-in edit variant in the future versions of MRT.

**Reply by tarasowski** [reply]
Thank you Kevin! Using custom Edit seems to be the perfect solution for me.
